github/src/main/kotlin/gropius/sync/github/JsonHelper.kt

```diff
@@ -0,0 +1,25 @@
+package gropius.sync.github
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.stereotype.Component
+
+/**
+ * Spring component containing helper function for github sync related json operations
+ * @param objectMapper Reference for the spring instance of ObjectMapper
+ */
+@Component
+class JsonHelper(
+    val objectMapper: ObjectMapper
+) {
+    /**
+     * Parse a json value into a string
```

Comment:
Maybe should explain in more detail what it does, that it only accepts a string and removes escape chars etc. if I understand correctly?

Reply:
It parses a json string into a string. I do not see the need for further explanation

Reply:
I just think "json value" is a bit ambiguous.
For me it can mean `{"field": "value\"containing\""}` or `"field": "value\"containing\""` or `"value\"containing\""` or `value\"containing\"` or `value"containing"`

Reply:
`@param input the json value stringified`

Reply:
if you have a better name for a plain, onobjected string after json encoding I would be happy to hear it

Reply:
Maybe a short example or an explanation that wrapping quotes should/shouldn't be included woud help...

Reply:
594aa1d062633212494306df27f3b24088743f6c

Reply:
Very nice, thank you